code/datums/medal_awards.dm

```diff
@@ -180,8 +276,9 @@ GLOBAL_LIST_INIT(human_medals, list(MARINE_CONDUCT_MEDAL, MARINE_BRONZE_HEART_ME
 		user.visible_message("ERROR: ID card not registered for [user.real_name] in USCM registry. Potential medal fraud detected.")
 		return
 
-	if(give_medal_award(get_turf(printer)))
-		user.visible_message(SPAN_NOTICE("[printer] prints a medal."))
+	GLOB.ic_medals_panel.user_locs[user] = printer
```

Comment:
You can't do that, that's DOUBLE reference fraud in a single line
Need to either setup deletion handlers or use weak_reference instead